src/Client.php

```diff
@@ -403,14 +460,4 @@ public function getStreamFactory(): StreamFactoryInterface
     {
         return $this->getHttpClientBuilder()->getStreamFactory();
     }
-
```

Comment:
Please revert this change.

Reply:
Reverted in d18e1d10fcd4f2768afcef81c011bcc29ce0c1e1